accounts-journey/src/androidTest/java/app_common/GenericInteractions.kt

```diff
@@ -0,0 +1,34 @@
+package app_common
+
+import androidx.test.espresso.ViewInteraction
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.closeSoftKeyboard
+import androidx.test.espresso.assertion.ViewAssertions.doesNotExist
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isClickable
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.isEnabled
+import androidx.test.espresso.matcher.ViewMatchers.withSubstring
+import androidx.test.espresso.matcher.ViewMatchers.withText
+
+fun ViewInteraction.shouldBeDisplayed(): ViewInteraction = check(matches(isDisplayed()))
+
+fun ViewInteraction.shouldMatchText(stringResource: Int): ViewInteraction =
+    check(matches(withText(stringResource)))
+
+fun ViewInteraction.shouldMatchText(expectedText: String): ViewInteraction = check(
+    matches(withText(expectedText))
+)
+
+fun ViewInteraction.shouldBeClickable(): ViewInteraction = check(matches(isClickable()))
+
+fun ViewInteraction.shouldContainText(expectedText: String): ViewInteraction = check(
+    matches(withSubstring(expectedText))
+)
+
+fun ViewInteraction.shouldNotExist(): ViewInteraction = check(doesNotExist())
+
+fun ViewInteraction.shouldBeEnabled(): ViewInteraction = check(matches(isEnabled()))
+
+fun ViewInteraction.typeText(text: String): ViewInteraction =
+    perform(ViewActions.typeText(text)).perform(closeSoftKeyboard())
```

Comment:
Could you add an import for the `ViewActions.typeText`, too?

Reply:
I had to rename the function to get rid of conflicts, but it is done